CHANGELOG.md

```diff
@@ -3,6 +3,177 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v6.0.0-alpha.3
+
+_Oct 13, 2022_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Simplify `useGridApiContext` and `useGridApiRef` type overrides (#6423) @cherniavskii
```

Comment:
Let me know if we should highlight something else too?

Reply:
I wouldn't highlight #6423, since it's an internal change and doesn't impact the users